<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>setupPose</class>
 <widget class="QDialog" name="setupPose">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>539</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>setupPose</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>초기 자세 설정</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startBtn">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>60</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>start</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="stopBtn">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>60</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>stop</string>
   </property>
  </widget>
  <widget class="QPushButton" name="captureBtn">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>60</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>capture</string>
   </property>
  </widget>
  <widget class="QWidget" name="camOut" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>481</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>startBtn</sender>
   <signal>clicked()</signal>
   <receiver>setupPose</receiver>
   <slot>onStartBtn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>277</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stopBtn</sender>
   <signal>clicked()</signal>
   <receiver>setupPose</receiver>
   <slot>onStopBtn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>406</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>captureBtn</sender>
   <signal>clicked()</signal>
   <receiver>setupPose</receiver>
   <slot>onCaptureBtn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>477</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>512</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onStartBtn()</slot>
  <slot>onStopBtn()</slot>
  <slot>onCaptureBtn()</slot>
 </slots>
</ui>
